core-backend/wasmi/src/state.rs

```diff
@@ -16,19 +16,18 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-use gear_backend_common::{BackendState, BackendTermination, TerminationReason};
-use gear_core::env::Ext;
+use gear_backend_common::{BackendExt, BackendState, BackendTermination, TerminationReason};
 use gear_core_errors::ExtError;
 
 pub type HostState<E> = Option<State<E>>;
 
-pub struct State<E: Ext> {
+pub struct State<E> {
```

Comment:
why not to paste here trait bound on E: BackendExt to clarify what's the purpose of the State type?

Reply:
Done as arkOf wants